django_lightweight_queue/runner.py

```diff
@@ -75,32 +76,34 @@ def handle_term(signum, stack):
                         worker.exitcode,
                     )
 
-                worker = Worker(
+                worker = subprocess.Popen([
+                    sys.executable,
+                    sys.args[0],
+                    'queue_worker',
                     queue,
-                    index,
-                    worker_num,
-                    running,
-                    log.level,
+                    str(worker_num),
+                    '--prometheus-port',
+                    str(app_settings.PROMETHEUS_START_PORT + index),
+                    '--log-level',
+                    logging._levelToName[log.level].lower(),
+                    '--log-file',
                     log_filename_fn('%s.%s' % (queue, worker_num)),
+                    '--touch-file',
                     touch_filename_fn(queue),
```

Comment:
It's a shame we have to use explicit option names here, vs inspecting them from queue_worker somehow.